test/web_console/helper_test.rb

```diff
@@ -23,6 +23,14 @@ def call(env)
         def request
           Request.new(@env)
         end
+
+        def status
```

Comment:
Do the tests pass without the status and the header? The console can be shown on regular pages as well. If we don _really_ need the status and the headers, I would cut them just so we have smaller diff.